src/NotesApplication/Models/Note.cs

```diff
@@ -15,6 +15,11 @@ public class Note
         /// </summary>
         internal string id { get; } //We should not allow updating of IDs because it could mess with the database
 
+        /// <summary>
+        /// Unique ID of the user
+        /// </summary>
+        internal string user_id { get; } //user does not change
```

Comment:
nit: change this to `owner` or `owner_id`. You can use the comments to describe what is actually is

Reply:
In the meeting on Friday we decided user id is not metadata(claims will store it) so I got rid of the field.